gst/isomp4/qtdemux.c

```diff
@@ -4684,7 +4684,7 @@ gst_qtdemux_chain (GstPad * sinkpad, GstBuffer * inbuf)
             /* only bother if it seems worth doing so,
              * and avoids possible upstream/server problems */
             if (demux->upstream_seekable &&
-                demux->upstream_size > 4 * (1 << 20)) {
```

Comment:
sorry, I think I'm going to change this line again

Reply:
it has a weird logic:
if upstream_size is bigger then 4G then seek. But we seek with a file of size 3 Gb